geoprocessing/src/migrations/geoprocessing/1614256630000-AddCountriesView.ts

```diff
@@ -0,0 +1,28 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddCountriesView1614256630000
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+CREATE UNIQUE INDEX only_countries_by_gid ON admin_regions(gid_0) WHERE (gid_0 IS NOT NULL AND gid_1 IS NULL AND gid_2 IS NULL);
```

Comment:
carefull that this dont get into the codebase as we discuss in pr #76

Reply:
yep, I'll have a conflict to fix when rebasing this on `develop` before merging 😅